params_windows.go

```diff
@@ -11,6 +11,10 @@ type PlatformSpecificParams struct {
 	// use the default ComponentId. The default ComponentId is set to tap0901,
 	// the one used by OpenVPN.
 	ComponentID string
+	// Of course, you may have multiple tap0901 adapters on the system, in which
+	// case we need a friendlier way to identify them. In that case we can use
+	// the friendly name of the network adapter as set in Control Panel.
```

Comment:
Please make this comment start with `InterfaceName`, and explain what this parameter is. Otherwise it won't pass linter.